core/src/main/java/org/infinispan/interceptors/distribution/TriangleDistributionInterceptor.java

```diff
@@ -91,12 +91,11 @@
    private static final Log log = LogFactory.getLog(TriangleDistributionInterceptor.class);
    private static final boolean trace = log.isTraceEnabled();
    private CommandAckCollector commandAckCollector;
+   private final InvocationExceptionFunction onPutMapNoLocalEntriesException = this::onPutMapNoLocalEntriesException;
```

Comment:
I think it's better to group the lambdas at the end then mixing them with the injected components.